Plugin/Api/CartSearchRepository.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+
+namespace Klaviyo\Reclaim\Plugin\Api;
+
+
+use Magento\Quote\Api\CartRepositoryInterface;
+use Magento\Quote\Api\Data\CartExtensionFactory;
+use Magento\Quote\Model\QuoteIdToMaskedQuoteId;
```

Comment:
what happens to customers on version pre 2.3?  If you look at https://github.com/magento/magento2/tree/2.2/app/code/Magento/Quote/Model it does not exist.

Reply:
we'll probably have to add this as a dependency